### ISSUE TITLE
iso-8859-1 encoding

### ISSUE THREAD

**Issue by schacon** [progit/progit2-es#2]
It looks like you're encoding all the files you're saving as iso-8859-1 somehow, which is breaking the build of the ES ebooks. I can re-encode them in UTF-8 but it's going to be an issue going forward unless you can fix it for new files you edit.

```
$ file -I 1-introduction.asc 
1-introduction.asc: text/plain; charset=iso-8859-1
```

Normal UTF-8 encoded files should look like this:

```
$ file -I 1-git-basics.asc 
1-git-basics.asc: text/plain; charset=utf-8
```

**Comment by andres-mancera** [comment]
I'll look into this.  Wil get back to you as soon as possible.

**Comment by andres-mancera** [comment]
@schacon : I have changed the encoding of all the files that I had checked in from iso-8859-1 to utf-8, so the ES ebook build should be fine now.  I am facing a problem though:  After the change, the text of some of my files is all garbled (for instance, this [one](https://github.com/progit/progit2-es/blob/master/book/01-introduction/sections/about-version-control.asc)).  The only thing that puzzles me is that some files are just fine (like this [one](https://github.com/progit/progit2-es/blob/master/book/01-introduction/1-introduction.asc)).  I will have to look into why this is happening.  If you have any suggestions, I am all ears.
